packages/terra-clinical-result/src/proptypes/statusPropTypes.js

```diff
@@ -0,0 +1,21 @@
+import PropTypes from 'prop-types';
+
+/**
+ * Enum for possible Result Statuses.
+ * values mapped from http //hl7.org/fhir/2018Sep/valueset-observation-status.html
+ */
+const statusPropOneOf = PropTypes.oneOf([
+  'registered',
+  'preliminary',
+  'final',
+  'amended',
+  'corrected',
+  'cancelled',
+  'entered-in-error',
+  'unknown',
+]);
+
+export default statusPropOneOf;
+export {
```

Comment:
Do we need both exports?

Reply:
Removed extraneous export https://github.com/cerner/terra-clinical/pull/574/commits/9eeacc2469da67d5f2d9ad3ce7cc533a407dbf13?file-filters%5B%5D=.js&file-filters%5B%5D=.json&file-filters%5B%5D=.jsx&file-filters%5B%5D=.scss&file-filters%5B%5D=.snap#diff-286f88d08c2a701946d93b9fa03faf71R18